integrations/maya/src/main/java/org/dash/wallet/integrations/maya/payments/MayaCryptoCurrency.kt

```
@@ -76,6 +82,19 @@ open class MayaBitcoinCryptoCurrency : MayaCryptoCurrency {
     override val addressParser: AddressParser = BitcoinAddressParser(BitcoinMainNetParams())
     override val codeId: Int = R.string.cryptocurrency_bitcoin_code
     override val nameId: Int = R.string.cryptocurrency_bitcoin_network
+    override fun getPaymentRequestURI(address: String, amount: String): String {
+        return try {
+            BitcoinURI.convertToBitcoinURI(
+                Address.fromBase58(BitcoinMainNetParams(), address),
+                Coin.parseCoin(amount),
+                null,
+                null
+            )
+        } catch (e: AddressFormatException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)